ensime_shared/protocol.py

```diff
@@ -247,3 +264,28 @@ def show_type(self, call_id, payload):
 
 class ProtocolHandlerV2(ProtocolHandlerV1):
     """Implements response handlers for the v2 ENSIME Jerky protocol."""
+
+    def handle_source_positions(self, call_id, payload):
+        """Handler for source positions"""
+        self.log.debug('handle_source_positions: in %s', Pretty(payload))
+
+        call_options = self.call_options[call_id]
+        word_under_cursor = call_options.get("word_under_cursor")
+
+        positions = payload["positions"]
+
+        if not positions:
+            self.editor.raw_message("No usages of <{}> found".format(word_under_cursor))
+            return
+
+        qfList = []
+        for p in positions:
+            position = p["position"]
+            preview = str(p["preview"]) if "preview" in p else "<no preview>"
+            item = self.editor.to_quickfix_item(str(position["file"]),
+                                                position["line"],
+                                                preview,
+                                                "info")
+            qfList.append(item)
+        qfSorted = sorted(qfList, key=itemgetter('filename', 'lnum'))
```

Comment:
Oh I gotcha! you mean camelCase vs snake_case.

I can look around and turn all names into snakes, maybe there's even a linter setting to ensure that. But again, would prefer to do it on a separate PR since this was not introduced in this PR